### ISSUE TITLE
ChatGPT groks FRBR

### ISSUE THREAD

**Issue by tombaker** [dcmi/openwemi#9]
I am posting the following with permission from Dan Brickley, who recently chatted with ChatGPT...

Dan:
Shakespeare wrote a book called Hamlet, so it is his. Macmillan's have a new edition of the final version of it out in the UK, so it is also somehow theirs too. I bought one and gave it to you, with a note written on page two, so this makes it yours.

 Please explain this confusing situation!

ChatGPT:
William Shakespeare wrote the play "Hamlet" and therefore he is considered the author of the work. The Macmillan's publishing company likely acquired the rights to publish a new edition of the play, which means they have the legal right to produce and distribute copies of the play. When you bought a copy of the play and gave it to someone else, you transferred ownership of that specific copy to them. However, the copyright of the work as a whole still belongs to the original author, Shakespeare, and the publishing company that holds the rights to publish it. So, while the person you gave the book to now owns that specific copy, Shakespeare and Macmillan's also have a claim to the work as a whole.

**Comment by tombaker** [comment]
Dan's follow-up question:
Use FRBR to explain
The Functional Requirements for Bibliographic Records (FRBR) is a conceptual model that is used to describe the relationships between different entities involved in the creation and dissemination of a work of art or literature. In the case of Shakespeare's play "Hamlet", the FRBR model would be used to describe the various roles and relationships involved in the creation and distribution of the play.

ChatGPT:
According to FRBR, there are four main entities involved in the production and dissemination of a work: the work itself, the expression of the work, the manifestation of the work, and the item of the work.

The work is the intellectual or artistic creation, in this case the play "Hamlet" as it was originally written by Shakespeare.
The expression of the work is the specific way in which the work is realized, such as a specific performance of the play or a specific edition of the script.
The manifestation of the work is the physical embodiment of the work, such as a printed copy of the script or a recording of a performance.
The item of the work is a specific copy of the work, such as a specific book or recording.
In the case of Shakespeare's play "Hamlet", the work is the play as it was written by Shakespeare. The expression of the work would be the final version of the play as it was published by Macmillan's. The manifestation of the work would be a physical copy of the play, such as the book you bought and gave to someone else. The item of the work would be that specific copy of the book, which is now owned by the person you gave it to.

In summary, FRBR describes the various entities involved in the creation and dissemination of a work, and the relationships between these entities. In the case of Shakespeare's play "Hamlet", FRBR helps to clarify the roles of Shakespeare as the author of the work, Macmillan's as the publisher of a specific edition of the play, and you and the person you gave the book to as the owners of a specific copy of the play.

**Comment by tombaker** [comment]
Dan:
Please write some Dublin Core RDF in Turtle to encode the basics of this situation.

ChatGPT:
The Dublin Core is a set of metadata elements that can be used to describe a variety of resources, including works of art and literature. The Resource Description Framework (RDF) is a language that can be used to encode Dublin Core metadata in a machine-readable format. Turtle is a popular syntax for writing RDF.

Here is an example of Dublin Core RDF in Turtle that could be used to encode the basics of the situation described in the previous question:

Copy code
@prefix dc: <http://purl.org/dc/elements/1.1/> .
@prefix ex: <http://example.com/work/> .

ex:work a dc:Work ;
    dc:title "Hamlet" ;
    dc:creator "William Shakespeare" ;
    dc:publisher "Macmillan's" ;
    dc:date "2022" .
This RDF uses the dc:Work class to indicate that the resource being described is a work of art or literature. The dc:title property is used to specify the title of the work, which in this case is "Hamlet". The dc:creator property is used to specify the author of the work, which in this case is "William Shakespeare". The dc:publisher property is used to specify the publisher of the work, which in this case is "Macmillan's". The dc:date property is used to specify the date the work was published, which in this case is "2022".

This RDF only encodes the basics of the situation described in the previous question. It does not include information about the specific edition of the play published by Macmillan's, or about the specific copy of the book that you bought and gave to someone else. To include this information, additional properties and classes from the Dublin Core and other vocabularies would need to be used.

**Comment by tombaker** [comment]
Dan:
Show the publication and my copy too.

ChatGPT:
To include information about the specific edition of the play published by Macmillan's, as well as information about the specific copy of the book that you bought and gave to someone else, we can add additional properties and classes to the Dublin Core RDF in Turtle. Here is an example of how this could be done:

@prefix dc: <http://purl.org/dc/elements/1.1/> .
@prefix ex: <http://example.com/work/> .

ex:work a dc:Work ;
    dc:title "Hamlet" ;
    dc:creator "William Shakespeare" ;
    dc:publisher "Macmillan's" ;
    dc:date "2022" .

ex:expression a dc:Expression ;
    dc:isPartOf ex:work ;
    dc:title "Hamlet: The Final Version" ;
    dc:publisher "Macmillan's" ;
    dc:date "2022" .

ex:manifestation a dc:Manifestation ;
    dc:isPartOf ex:expression ;
    dc:title "Hamlet: The Final Version" ;
    dc:publisher "Macmillan's" ;
    dc:date "2022" ;
    dc:format "Book" .

ex:item a dc:Item ;
    dc:isPartOf ex:manifestation ;
    dc:title "Hamlet: The Final Version" ;
    dc:publisher "Macmillan's" ;
    dc:date "2022" ;
    dc:format "Book" ;
    dc:provenance "Given to X by Y" .

**Comment by danbri** [comment]
Nifty eh? It’s worth getting an account and trying it out. Remembering you can pass it longish text as a prompt.

**Comment by kcoyle** [comment]
ChatGPT's explanation makes way more sense than IFLA's. I wonder what it would do with the non-book examples in my [Code4lib article](https://journal.code4lib.org/articles/16491) - or any other uses beyond books.

**Comment by danbri** [comment]
> The Code4Lib Journal is a publication that covers topics related to technology and libraries. The article "Works, Expressions, Manifestations, Items: An Ontology" discusses the concepts introduced in the FRBR document, which are known as "WEMI" (Work, Expression, Manifestation, Item). The article argues that these concepts, which were originally developed for use in library bibliographic catalogs, have been useful in other contexts as well, and proposes a set of classes and relationships that could form the basis for a more general model of created works

**Comment by danbri** [comment]
> The article proposes a set of classes and relationships that can be used as a model of created works. This model is based on the concepts introduced in the FRBR document, which are known as "WEMI" (Work, Expression, Manifestation, Item). These concepts were originally developed for use in library bibliographic catalogs, but have been found to be useful in other contexts as well. For example, the WEMI concepts have been used in the fashion industry to describe the design process, and in the legal field to track the evolution of legal documents. The proposed model is intended to be a more general version of the WEMI concepts that is not tied to the specific use case of library catalogs.
